testing_and_setup/compass/ocean/hurricane/USDEQU120at30cr10rr2/synthetic_sandy/config_init.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0"?>
+<config case="init">
+	<add_link source="../../build_mesh/culled_mesh/culled_mesh.nc" dest="mesh.nc"/>
+	<add_link source="../../build_mesh/culled_mesh/culled_graph.info" dest="graph.info"/>
+	<add_link source_path="script_configuration_dir" source="create_pointstats_file.py" dest="create_pointstats_file.py"/>
+	<add_link source_path="script_configuration_dir" source="create_pointstats_file.py" dest="create_pointstats_file.py"/>
+	<add_link source_path="script_test_dir" source="NOAA-COOPS_stations.txt" dest="NOAA-COOPS_stations.txt"/>
```

Comment:
`script_test_dir` were broken links for me.

Reply:
I think you want `script_configuration_dir` instead.

Reply:
Actually, there are unneeded files here that can be removed.

Reply:
I fixed the broken link issue. Thanks for catching that!

Reply:
We still need the `NOAA-COOPS_stations.txt` and `USGS_stations.txt` files to specify the station locations in the pointstats input. These files just contain the station locations, not observed data (which we don't need).

Reply:
Fixed now.

Reply:
I agree, this is fixed now.